packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-const-for-loop/actual.js

```diff
@@ -0,0 +1,2 @@
+const j = 0;
+for (const x = 0; ; ) console.log(j + x);
```

Comment:
move this to previous test - as it tests block scoped vars merged to for loop

Reply:
cool